ci/jenkins/jobs/macros.yaml

```diff
@@ -72,7 +72,15 @@
                   --vcuser "${VC_USER}" --datacenter "${DATACENTERNAME}" --datastore "${DATA_STORE}" \
                   --vcCluster "${VC_CLUSTER}" --resourcePool "${RESOURCEPOOLPATH}" --vcNetwork "${VMC_NETWORK_1}" \
                   --virtualMachine "${VIRTUAL_MACHINE}" --goVcPassword "${GOVC_PASSWORD}" --testType aks-with-agent
-
+- builder:
+    name: deploy-windows-vm-with-agent-and-run-eks-tests-for-nephe
+    builders:
+      - shell: |-
+          #!/bin/bash
+          ./ci/jenkins/nephe-ci.sh --buildnumber "${BUILD_NUMBER}" --vchost "${VC_HOST}" \
+                  --vcuser "${VC_USER}" --datacenter "${DATACENTERNAME}" --datastore "${DATA_STORE}" \
+                  --vcCluster "${VC_CLUSTER}" --resourcePool "${RESOURCEPOOLPATH}" --vcNetwork "${VMC_NETWORK_1}" \
+                  --virtualMachine "${VIRTUAL_MACHINE}" --goVcPassword "${GOVC_PASSWORD}" --testType eks-with-windows
```

Comment:
-with-windows, doesnt indicate its agented mode

Reply:
Changed it to -eks-with-windows-agent